examples/showcase/tracked-controls/js/components/collider.js

```diff
@@ -0,0 +1,109 @@
+/* global AFRAME, THREE */
+
+/**
+ * It implements AABB collision detection for entities with a mesh.
```

Comment:
nit: could tighten comments a bit by removing the `It`, just state what the code does: "Implement AABB collision..."